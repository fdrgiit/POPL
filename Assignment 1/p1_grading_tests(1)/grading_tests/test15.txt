 
 x111abcdefg4444492359.111111111111111111123452354235346456457091240234real
 
 
