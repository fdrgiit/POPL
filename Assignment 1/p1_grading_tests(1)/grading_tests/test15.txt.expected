2 37 x111abcdefg4444492359 
2 33 .
2 34 111111111111111111123452354235346456457091240234 
2 37 real 
